templates/web/greenwich/front/_list-entry.html

```diff
@@ -0,0 +1 @@
+../../oxfordshire/front/_list-entry.html
```

Comment:
I'm not massively keen on introducing dependencies between cobrands like this, as I know it'll catch me out in the future... Given it's a small template could it just be duplicated here?